docs/installation/updating-with-npm.md

```diff
@@ -18,26 +16,19 @@ If you do not have command line access, you can see the version number in the
 
 ```json
 'govuk-frontend': ‘2.11.0’
-````
+```
 
 ## Update
 
 You can check the [release
-notes](https://github.com/alphagov/frontend/releases) in the GOV.UK Frontend GitHub repository to find out the latest version of GOV.UK Frontend.
-
-You can update to any version.
-
-You may need to make code changes to make sure GOV.UK Frontend keeps working in your project, if you’re updating either:
-
-- to a major version - for example you’re updating to 2.0.0
-- past a major version - for example you’re updating from 1.2.0 to 2.3.0
+notes](https://github.com/alphagov/govuk-frontend/releases) in the GOV.UK Frontend GitHub repository to find out the latest version of GOV.UK Frontend.
 
-A version is major ('breaking') if there's a 'Breaking changes' section in that version's release notes.
+You may need to make code changes to keep GOV.UK Frontend working in your project, if the major version number will change when you update. The major version number is the first digit in the version number.
 
 To update to the most recent version, run:
 
 ```shell
 npm install govuk-frontend@latest
 ```
 
-If you want to install a specific version, replace `@latest` with the version that you want to update to. For example `2.13.0`.
+If you want to install an earlier version, replace `latest` with the version that you want to update to. For example `2.13.0`.
```

Comment:
@hannalaakso In retrospect I thought it was best in the end to remove the 'You can update to any version' line. Because with a tweak to this line 34 - to say "earlier" rather than "specific", I think we cover off the idea that you can update to any version?

Reply:
@m-green I'm happy with that 👍